public/locales/en_US/translation.json

```diff
@@ -435,6 +435,9 @@
 	"No Researchers. Add Researchers by clicking the [+] button above.": "No Researchers. Add Researchers by clicking the [+] button above.",
 	"Are you sure you want to delete this Researcher?": "Are you sure you want to delete this Researcher?",
 	"Well done!": "Well done!",
+	"Swipe around the": "Swipe around the",
+	"screen to reveal": "screen to reveal",
+	"the hidden image": "the hidden image",

```

Comment:
Why is this message broken up into three parts?

Reply:
They are inside 3 different HTML components. that is why.